<launch>
	
    <arg name="soft_top_speed" default="10.0"/>
    <arg name="a_max_horizontal" default="6.5"/>
    <arg name="yaw_on" default="true"/>
    <arg name="use_depth_image" default="false"/>
    <arg name="max_acceleration_total" default="7.5"/>
    <arg name="min_speed_at_max_acceleration_total" default="10.0"/>
    <arg name="flight_altitude" default="3.0"/>
  
    <param name="soft_top_speed" type="double" value="$(arg soft_top_speed)"/>
	<param name="a_max_horizontal" type="double" value="$(arg a_max_horizontal)"/>
	<param name="yaw_on" type="bool" value="$(arg yaw_on)"/>
    <param name="use_depth_image" type="bool" value="$(arg use_depth_image)"/>
    <param name="max_acceleration_total" type="double" value="$(arg max_acceleration_total)"/>
    <param name="min_speed_at_max_acceleration_total" type="double" value="$(arg min_speed_at_max_acceleration_total)"/>
    <param name="flight_altitude" type="double" value="$(arg flight_altitude)"/>


    <node pkg="motion_primitives" type="motion_selector_node" name="motion_selector" output="screen">
        <remap from="/local_goal" to="/move_base_simple/goal" />
    </node>
   
</launch>
